include/grpc/support/avl.h

```diff
@@ -57,25 +62,27 @@ typedef struct gpr_avl {
 GPRAPI gpr_avl gpr_avl_create(const gpr_avl_vtable *vtable);
 /** add a reference to an existing tree - returns
```

Comment:
We should update the docstring for all the modified functions that now take `user_data`